src/Collector/HttpStreamCollector.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Debug\Collector;
+
+final class HttpStreamCollector implements CollectorInterface, IndexCollectorInterface
+{
+    use CollectorTrait;
+
+    public function __construct(private array $ignoredPathPatterns = [], private array $ignoredClasses = [])
+    {
+    }
+
+    private array $requests = [];
+
+    public function getCollected(): array
+    {
+        return $this->requests;
+    }
+
+    public function startup(): void
+    {
+        $this->isActive = true;
+        HttpStreamProxy::register();
+        // TODO: add cURL support, maybe through proxy?
```

Comment:
Do you plan to implement it in this PR? If you plan it for later, please create an issue about it.

Reply:
Unfortunately, It's impossible to do it now because PHP doesn't support it

Reply:
https://github.com/yiisoft/yii-debug/issues/192